src/entity/Menu.java

```diff
@@ -0,0 +1,74 @@
+package entity;
+
+import java.io.Serializable;
+
+public class Menu implements Serializable, Comparable<Menu> {
+    private Integer id;
+    private String name;
+    private String description;
+    private double price;
+    private String image;
+    private int parentId;
+    private boolean isDeleted;
+
+    public Integer getId() {
+        return id;
+    }
+
+    public void setId(Integer id) {
+        this.id = id;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
+
+    public String getDescription() {
+        return description;
+    }
+
+    public void setDescription(String description) {
+        this.description = description;
+    }
+
+    public double getPrice() {
+        return price;
+    }
+
+    public void setPrice(double price) {
+        this.price = price;
+    }
+
+    public String getImage() {
+        return image;
+    }
+
+    public void setImage(String image) {
+        this.image = image;
+    }
+
+    public int getParentId() {
+        return parentId;
+    }
+
+    public void setParentId(int parentId) {
+        this.parentId = parentId;
+    }
+
+    public boolean isDeleted() {
+        return isDeleted;
+    }
+
+    public void setDeleted(boolean deleted) {
+        isDeleted = deleted;
+    }
+
+    @Override
+    public int compareTo(Menu menu) {
+        return this.getId().compareTo(menu.getId());
```

Comment:
Why did you compare the menu based on ID?

Reply:
Because the id is unique